projects/libtiff/build.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash -eu
+#
+# Copyright 2023 ISP RAS
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
```

Comment:
fix license
Leave copyright as here: https://gitlab.com/libtiff/libtiff/-/blob/master/contrib/oss-fuzz/build.sh
Add modifications copyright for ISP RAS

You can find example in lua/build.sh header